src/web/ballot.js

```diff
@@ -263,9 +263,12 @@ export function buildUI({ ethereumAddress, getElementById, fetch, now }) {
         return html``;
       }
       const { label, info } = state.event;
-      if (!info || label === 'RESULT') {
+      if (!info ) {
         return html`${label}`;
       }
+      if ( label === 'RESULT') {
+        return html`${"VOTE SUCCESSFULLY REGISTERED !"}`;
```

Comment:
I'm not a fan of SHOUTING. I suggest:

```suggestion
        return html`${"Vote successfully registered"}`;
```